File: service-invocation/java/sdk/frontend/src/main/java/com/dapr/sdk/Controllers/ApiController.java
package com.dapr.sdk.Controllers;

import java.lang.reflect.Array;
import java.util.ArrayList;
import java.util.List;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

import io.dapr.client.DaprClient;
import io.dapr.client.DaprClientBuilder;
import io.dapr.client.domain.HttpExtension;
import reactor.core.publisher.Mono;

@RestController
public class ApiController {
    String DAPR_BACKEND_SERVICE = "backend";
    String DAPR_BACKEND_METHOD = "api/get/cars";

    @GetMapping("/api/dapr")
    public Mono<List> apiController() throws Exception {
        try (DaprClient client = (new DaprClientBuilder()).build()) {
            Mono<List> apiResponse = client
                    .invokeMethod(DAPR_BACKEND_SERVICE, DAPR_BACKEND_METHOD, null, HttpExtension.GET, List.class);

            System.out.println(apiResponse);
            return apiResponse;
        }
    }
}

File: service-invocation/php/sdk/frontend/app/Http/Controllers/ApiController.php
<?php

namespace App\Http\Controllers;

use Dapr\Client\DaprClient;
use Dapr\Client\AppId;

class ApiController extends Controller
{
    public function apiController()
    {
        $client = DaprClient::clientBuilder()->build();
        $appId = new AppId('backend');
        $res = $client->invokeMethod('GET', $appId, "api/cars/get")
        ->getBody()
        ->getContents();
        echo($res);
        return view('api', ['data' => json_encode($res)]);
    }
}